packages/api/src/command/connected-user/get-connected-user.ts

```diff
@@ -55,6 +55,14 @@ export const getConnectedUsers = async ({
   });
 };
 
+export const getAllConnectedUsers = async (cxId?: string): Promise<ConnectedUser[]> => {
```

Comment:
I only realized this now: this function is dangerous, someone in the future could call it while building/fixing something and end up returning all users to a given cx.

Other functions have `cxId` as required, so its not as risky (not suggesting to change this, though).

Maybe we can keep this on the route itself? We shouldn't have much use for it besides that route, or other admin ones - in which case we can reconvene.

If you feel strong about it, I think at the very least we should flag this with a comment flagging this is only for ops/admin purposes.